### DISCUSSION TITLE
Series not appearing in search

### DISCUSSION THREAD

**Discussion by crank8877** [comictagger/comictagger#707] [Q&A] [answered]
I'm having trouble getting the tags for the 1968 Nick Fury, Agent of SHIELD. I've confirmed with ComicVine that the title and year are all correct, but it doesn't even appear in the search results. As you see in this screenshot where the results are sorted by year, that isn't one of the results.

ETA: Seems to be a problem with "Nick Fury" in general. The 2017 Nick Fury series also won't appear in searches.

![Screenshot 2024-12-08 at 8 56 33 (2)](https://github.com/user-attachments/assets/8789996b-cd04-4e32-9fc1-ad1bf49a76be)

**Comment by lordwelch** [comment]
Works fine for me
<img width="1470" alt="Screenshot 2024-12-07 at 4 36 31 PM" src="https://github.com/user-attachments/assets/a568e8a9-dcd9-4901-afc4-e2776b569cc3">
Can you post the log?. You can open the settings `Cmd+,` and click on the log directory to open it

**Reply by crank8877** [reply]
Weird, doesn't show in my searches at all. Log is attached.

[ComicTagger.log](https://github.com/user-attachments/files/18050350/ComicTagger.log)

**Reply by crank8877** [reply, marked as answer]
Seems to have been an issue with the cache. I saw the option to clear it in the settings, gave it a shot, and now the results have appeared.
